public/externalLibs/sound/sounds.js

```diff
@@ -308,14 +306,18 @@ var _safeaudio = null;
  */
 function play(sound) {
     // If a sound is already playing, terminate execution.
-    if (_safeplaying || _playing) return;
-    // Discretize the input sound
-    var data = discretize(get_wave(sound), get_duration(sound));
-    _safeaudio = raw_to_audio(data);
-
-    _safeaudio.addEventListener('ended', stop);
-    _safeaudio.play();
-    _safeplaying = true;
+    if (_safeplaying || _playing) {
+        throw new Error("play: audio system still playing previous sound");
+    } else {
+        // Discretize the input sound
+        var data = discretize(get_wave(sound), get_duration(sound));
+        _safeaudio = raw_to_audio(data);
+
+        _safeaudio.addEventListener('ended', stop);
+        _safeaudio.play();
+        _safeplaying = true;
+        return sound;
```

Comment:
The doc says it's supposed to return `undefined`, but this isn't undefined.

Reply:
The PR is supposed to change the specs for some functions (e.g. ADSR now accepts ratios instead of durations)

Reply:
Where can I update the doc?

Reply:
The jsdoc of this function

Reply:
Changed the jsdoc. Does the official source documentation pull the info from the academy frontend or is it static?